lib/Runtime/Library/TypedArray.cpp

```diff
@@ -46,6 +46,44 @@ namespace Js
     {
     }
 
+    Var TypedArrayBase::CreateNewInstanceFromObj(RecyclableObject *object, ScriptContext *scriptContext, uint32 elementSize, PFNCreateTypedArray pfnCreateTypedArray)
+    {
```

Comment:
There is similar in the TypedArrayBase::EntryFrom - which creates from iterable object - could we refactor and use only one implementation?